### DISCUSSION TITLE
yubikey support

### DISCUSSION THREAD

**Discussion by MorgothSauron** [standardnotes/forum#2681] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes.

**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
Allow to use Yubikey as a 2FA option and allow to configure more than one key. 

**Describe alternatives you've considered**
I already use the existing 2FA option, but I would prefer to have Yubikey as an option.

**Additional context**
Other privacy services do propose to use yubikey (Proton, Tutanota, Bitwarden) as a 2FA option.

**Comment by quirkyheadband** [comment]
I would like to use Standard Notes as my primary TOTP application, but I don't really want to do that if I have to use another TOTP application to get into Standard Notes.

**Reply by MikeGreat** [reply]
> I would like to use Standard Notes as my primary TOTP application, but I don't really want to do that if I have to use another TOTP application to get into Standard Notes.

+1 for this use case.

I, on the other hand, use SN as my primary TOTP authenticator.

**Comment by sparky-73** [comment]
Yes, Yubikey support would be a welcomed feature!

**Comment by cyanide699** [comment]
+1 would love to see Fido/webauth implemented

**Comment by cyanide699** [comment]
+1

**Comment by itinfosec** [comment]
Would love YubiKey support
